app/views/content/life-as-a-teacher/age-groups-and-specialisms/special-educational-needs.md

```diff
@@ -17,32 +17,82 @@ keywords:
+  
+* teaching in a special school
+* becoming a special educational needs co-ordinator (SENCO)
+* completing the mandatory qualification in sensory impairment (MQSI)
+
+<blockquote class="quote quote--background-white quote--indent">
+  <p>My mentor was the special educational needs co-ordinator. This meant I was able to learn more about this role and improve my understanding of the strategies and support available for children with special educational needs and disabilities."</p>
+    <footer class="footer">
+      <div class="author">
+        <div>
+            <cite class="name">Abbe, trainee teacher</cite>
+        </div>
+      </div>
+</footer>
+</blockquote>
+
+
+### Teaching in a special school
+
+Special schools provide education for pupils with special educational needs and disabilities. Special schools with pupils aged 11 and older can specialise in 1 of the 4 areas of special educational needs: 
+
+* communication and interaction 
+
+* cognition and learning 
+
+* social, emotional and mental health 
+
+* sensory and physical needs 
 
-No matter what settings you train and work in – whether it's mainstream schools, special schools, or both – you’ll work with pupils with complex needs.
+During your initial teacher training, you may have the opportunity to have one of your school placements in a special school. This can enhance your understanding of how to plan and teach more effectively for all children.  
 
-$q-jane-wilkinson$
+### Becoming a special educational needs co-ordinator
 
-You can [find a teacher training course](https://find-teacher-training-courses.service.gov.uk/) and filter by 'courses with a SEND specialism' if you have a particular interest in the area.
+A special educational needs coordinator (SENCO) assesses, plans and monitors the progress of pupils with special educational needs and disabilities in mainstream schools. SENCOs also support teachers and teaching assistants in a range of ways including providing training, observing lessons and developing learning plans for pupils. 
 
-However, it’s not essential to do a course specialising in SEND.
+As a SENCO you will get to know the specific needs of pupils who need extra support, so you can help them to feel comfortable and confident before teaching starts. 
 
-As you train and teach you'll get the skills you need to work with disabled pupils and pupils with special educational needs.
+To become a SENCO, you’ll need to gain [qualified teacher status (QTS)](/train-to-be-a-teacher/what-is-qts) through initial teacher training and then complete a mandatory [SENCO national professional qualification (SENCO NPQ)](https://www.gov.uk/guidance/special-educational-needs-co-ordinators-national-professional-qualification). You do not need to complete the NPQ before taking a SENCO role, you have 3 years to complete the qualification. 
```

Comment:
The last sentence is slightly confusing to me (not a fan of commas used like this but that could just be me!). I'm not sure how the 3 years quite relates - is it 3 years from taking a SENCO role? 3 years from starting the NPQ before becoming a SENCO?

Would it be clearer to break it into two sentences along the lines of:
`You do not need to complete the NPQ before taking a SENCO role. Once you become a SENCO, you must complete the qualification within 3 years.`

(I don't know if this is accurate - would need checking!)